frontend/timesheet/timesheet.scss

```diff
@@ -206,18 +226,38 @@
     padding: 10px;
 }
 
+.time-mask-hours{
+    padding-right: 21px;
+}
+
 .timesheet-hours{
     width: 100%;
     height: 47px;
     margin: 0 auto;
-    text-align: center;
+    text-align: right;
     @include hide-control-border();
 }
 
+.timesheet-key{
+    float: left;
+}
+
+.table .timesheet-project-filter{
+    float: right;
+}
+.timesheet .report-filter{
+    margin: -3px;
+}
+
+.clear-both{
```

Comment:
Never ever make a class that repeats the css style. Remove that class and use css property where you need to clear both.